src/core/app.lua

```diff
@@ -517,8 +517,11 @@ function breathe ()
       end
    end
    counter.add(breaths)
-   -- Commit counters at a reasonable frequency
-   if counter.read(breaths) % 100 == 0 then counter.commit() end
+   -- Commit counters and rebalance freelists at a reasonable frequency
```

Comment:
Do you have hydra comparisons that this additional rebalance_freelists call is perf-neutral?  If it would be possible to put this call in the pull or push function of an interlink app, that would be better, of course -- is it possible?  (Maybe not.)